src/RabbitMQAttribute.cs

```diff
@@ -29,6 +29,17 @@ public sealed class RabbitMQAttribute : Attribute
         [AutoResolve]
         public string QueueName { get; set; }
 
+        [AutoResolve]
+        public string UserName { get; set; }
+
+        [AutoResolve]
+        public string Password { get; set; }
+
+        // Optional
+        public int Port { get; set; }
+
+        public string ConnectionStringSetting { get; set; }
+
         [AutoResolve]
         public string Exchange { get; set; }

```

Comment:
Is exchange optional? If yes, add documentation in summary tags with info on the default value

Reply:
Removing exchange for now and setting default value to the empty string